docs/sources/alerting/set-up/provision-alerting-resources/file-provisioning/index.md

```diff
@@ -138,12 +138,12 @@ deleteRules:
 Create or delete contact points in your Grafana instance(s).
 
 1. Create a contact point in Grafana.
-1. Use the [Alerting provisioning API]({{< relref "../../../../developers/http_api/alerting_provisioning" >}}) export endpoints to download a provisioning file for your contact point.
-1. Copy the contents into a YAML or JSON configuration file in the default provisioning directory or in your configured directory.
+2. Use the [Alerting provisioning API][alerting_provisioning] export endpoints to download a provisioning file for your contact point.
```

Comment:
We should prefer lazy listing if possible for automatic numbering.

Reply:
done

Reply:
I don't think it is, sorry.